### PR TITLE
add tags input for text editor

### PR DESCRIPTION
**Pull request by mvoloshin0458**
I'll do more changes to the TextEditor component in the scope of diagnostics page, sp didn't test in wekapp yet. I'm going to test after finishing the page

### CODE REVIEW

File: src/components/TextEditor/components/TagsInput/TagsInput.tsx
Comment (sofa-tata) [inline src/components/TextEditor/components/TagsInput/TagsInput.tsx:1]: there is this `TagsBox` component, with tags validation and error display etc
Feeling bad to say it after you've done such significant job creating this one 😢

Comment (mvoloshin0458) [reply src/components/TextEditor/components/TagsInput/TagsInput.tsx:1]: will need to check more carefully next time

File: src/components/ExpandCollapseButton/ExpandCollapseButton.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  shouldCollapse: boolean
+  disabled?: boolean
+}
+
+function ExpandCollapseButton(props: ExpandCollapseButtonProps) {
+  const { onChange, shouldCollapse, disabled } = props
+
+  return (
+    <Tooltip
+      data={disabled ? '' : shouldCollapse ? 'Expand All' : 'Collapse All'}
```
Comment (sofa-tata) [inline src/components/ExpandCollapseButton/ExpandCollapseButton.tsx:20]: you can use `EMPTY_STRING`

File: src/components/TextEditor/components/FoldAllButton/FoldAllButton.tsx [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
-      shouldCollapse={!!shouldFoldAll}
-      onChange={(newVal) =>
-        setTextEditorContext((prev) => ({
-          ...prev,
-          shouldFoldAll: newVal
-        }))
-      }
-    />
+    !hidden && (
+      <Tooltip data={disabled || ''}>
```
Comment (sofa-tata) [inline src/components/TextEditor/components/FoldAllButton/FoldAllButton.tsx:23]: also `EMPTY_STRING`:)

**Review by sofa-tata — APPROVED** [review]

**Pull request merged**